helper.go

```diff
@@ -0,0 +1,40 @@
+package ipldgit
+
+import (
+	"bytes"
+	"compress/zlib"
+	"errors"
+	"io"
+
+	blocks "github.com/ipfs/go-block-format"
+	"github.com/ipfs/go-cid"
+	"github.com/ipld/go-ipld-prime"
+	mh "github.com/multiformats/go-multihash"
+)
+
+// DecodeBlock attempts to parse a serialized ipfs block into an ipld node dag
+func DecodeBlock(block blocks.Block) (ipld.Node, error) {
```

Comment:
I assume we want to keep this for backwards compatibility. Do we want to at least deprecate it?